sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/BarcodableViewDao.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.bbsrc.tgac.miso.persistence;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Barcodable;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.view.BarcodableView;
+
+import java.util.Collection;
+import java.util.List;
+
+//public interface BarcodableViewDao extends PaginatedDataSource<BarcodableView> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)